src/hook/useAuth.tsx

```diff
@@ -54,9 +58,14 @@ const useAuth = () => {
   };
 
   const handleSignOut = async () => {
-    await auth()
-      .signOut()
-      .then(() => Alert.alert('Información', 'Se ha cerrado la sesión'));
+    try {
+      await GoogleSignin.signOut();
+      await auth()
+        .signOut()
+        .then(() => Alert.alert('Información', 'Se ha cerrado la sesión'));
+    } catch (error) {
+      console.error(error);
```

Comment:
don't use console.log() use Alert, console.log is only for look errors on development